src/redux/selectors.js

```diff
@@ -1,3 +1,23 @@
+import { isWithinRange } from 'date-fns';
+import { find } from 'lodash';
+
+const getVelocity = (stories, sprint) =>
```

Comment:
From looking at the b2io/scopebuilder + AEP/TLE report, it appears there may be an issue with calculating velocity for the first sprint when there were stories closed before it started. The 0th tick on the graph shows 198 closed, the 1st tick shows 221 closed, but the report lists 17 as the velocity for Sprint 1 rather than 23. 

![image](https://user-images.githubusercontent.com/492553/35922130-c2c3e216-0bea-11e8-8f85-1784b526762d.png)

Reply:
Looking further in, there's issues that closed on the day the sprint ended that aren't counted as being in that sprint.

![image](https://user-images.githubusercontent.com/492553/35922779-72cad3d0-0bec-11e8-80d6-ed32fd50940a.png)

![image](https://user-images.githubusercontent.com/492553/35922873-b96dc216-0bec-11e8-8cb9-440fb2c668fc.png)

That six points would account for the 17 to 23 offset. This probably is a start-of-day alignment issue looking at it.

Reply:
⚠️: This also makes me wonder about an edge-case where a story is closed _between_ sprints when the sprints don't include the weekends (like in b2io/scopebuilder).